### PR TITLE
Remove dependencies of unzip and juju from host machine

### PR DESCRIPTION
**Pull request by addyess**

### CODE REVIEW

File: shrinkwrap-lxc.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ sleep 2
 
 function dependencies {
   lxc exec shrinkwrap -- apt update
-  lxc exec shrinkwrap -- apt install -y python3-pip unzip docker.io
+  lxc exec shrinkwrap -- apt install -y python3-pip docker.io
   lxc exec shrinkwrap -- pip install -r /root/requirements.txt
   lxc exec shrinkwrap -- snap install juju --classic
```
Comment (kwmonroe) [inline shrinkwrap-lxc.sh:14]: We can drop this too, right?

**Review by kwmonroe — COMMENTED** [review]
Needs tweaking since #18 went in.  Minor inline question for the lxc script.

File: shrinkwrap-lxc.sh (hunk 1 above)
Comment (addyess) [reply shrinkwrap-lxc.sh:14]: drop docker?  I think i'm still using `docker pull/docker save` to create `.tgz` images of the containers

If there's another way i should pull them to tgz without using docker -- that'd be nice.

Comment (addyess) [reply shrinkwrap-lxc.sh]: Done

**Review by Cynerva — APPROVED** [review]

**Pull request merged**